PBN RCT.Rmd

```diff
@@ -0,0 +1,544 @@
+---
+title: "PBN RCT"
+author: "Arielle Smith"
+date: "2024-04-24"
+output:
+  html_document: default
+  pdf_document: default
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+#Load required packages
+if(!require(tidyverse)){install.packages('tidyverse')}
+library(tidyverse)
+if(!require(lmtest)){install.packages('lmtest')}
+library(lmtest)
+if(!require(car)){install.packages('car')}
+library(car)
+if(!require(olsrr)){install.packages('olsrr')}
+library(olsrr)
+if(!require(sandwich)){install.packages('sandwich')}
+library(sandwich)
+if(!require(MOTE)){install.packages('MOTE')}
+library(MOTE)
+
+#Set working directory
+setwd("/Volumes/fsmresfiles/MSS/Schleider_Lab/jslab/Project Body Neutrality/RCT/Data/Full Sample (Baseline)")
+
+#Load data
+original_survey_data <- read.csv("original_survey_data.csv")
+new_survey_data <- read.csv("new_survey_data.csv")
+```
+
+```{r merge and clean data}
+#Merge original_survey_data (closed because of frauds) and new_survey_data
+
+##original_survey_data has 731 columns but new_survey_data has 729. Figure out the 2 differences
+cols_in_original_only <- setdiff(names(original_survey_data), names(new_survey_data))
+print(cols_in_original_only)
+
+##Remove the 2 extra Qualtrics columns from original_survey_data (won't mess things up because they are at the end and don't have anything important)
+original_survey_data <- original_survey_data %>%
+  select(-Q_AmbiguousTextPresent, -Q_AmbiguousTextQuestions)
+
+##Now we can merge
+merged_data <- rbind(original_survey_data, new_survey_data)
+
+#Create a binary variable for Condition# where 0 = Control and 1 = Intervention
+merged_data <- merged_data %>%
+  mutate(Condition = case_when(
+    Condition == "Control" ~ 0,
+    Condition == "Intervention" ~ 1))
+
+#Select the participants being included in analysis (per data integrity checks, should be N=220)
+included_participants <- merged_data %>%
```

Comment:
Yes that's right. I filtered the participant tracking spreadsheet by who we included, and only copy and pasted those IDs. The problem is that the inclusion decisions changed overtime and we made subjective decisions at times when we were on the fence, so I don't think it will be possible to write code to identify the right people. I do feel confident though that this is accurate.